draft-ietf-gnap-core-protocol.md

```diff
@@ -5419,6 +5419,68 @@ use polling to continue the request. The tradeoffs of this approach are discusse
 {{security-polling}}, and if possible, an explicit interaction finish method should be
 used instead.
 
+## Calculating Interaction Hash {#security-interact-hash}
+
+The calculation of the interaction hash value provides defence in depth, allowing a client
+instance to protect itself from spurious injection of interaction references when using a
+callback finish method. An attacker attempting to substitute the interaction reference
+is stopped in several places.
```

Comment:
I think it'd be worth calling out the main way injection is prevented (by the AS) here as well.